### ISSUE TITLE
Add option custom path to sitemanager.xml

### ISSUE THREAD

**Issue by Arttse** [timomeh/filezilla-sitemanager-cli#5]
Option custom path to `sitemanager.xml` would be great.

**Comment by timomeh** [comment]
Yeah, thought of that too while I programmed the first version. It's a bit wobbly to have it hard-coded in there.

What do you think:
- should it be an environment variable for the whole terminal session in something like a `.bash_profile`:  
  `export FILEZILLA_SITEMANAGER_PATH=~/.sitemanager.xml`  
  This way a user could choose the path globally when using `fsc`
- or should it be an option in the `fsc` command, like:  
  `fsc --path ~/sitemanager.xml`  
  This way it would be just for the current command
- or ![why_not_both-](https://cloud.githubusercontent.com/assets/4227520/13894739/b3b6235a-ed6b-11e5-94b3-39295eb57eb8.jpg)
